
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/bee1/wiibee/wiiboard.py", line 152, in loop
    self.battery = data[14] * 0.039315 + 0.0907875 
IndexError: index out of range
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/bee1/wiibee/wiiboard.py", line 152, in loop
    self.battery = data[14] * 0.039315 + 0.0907875 
IndexError: index out of range
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/bee1/wiibee/wiiboard.py", line 152, in loop
    self.battery = data[14] * 0.039315 + 0.0907875 
IndexError: index out of range
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/bee1/wiibee/wiiboard.py", line 152, in loop
    self.battery = data[14] * 0.039315 + 0.0907875 
IndexError: index out of range
